spec/features/create_posts_spec.rb

```diff
@@ -6,6 +6,12 @@
     expect(current_path).to match "edit"
   end
 
+  scenario 'creating a blog with a different language then english' do
+    create_a_blog_post(:language => "ro")
+    visit admin_blog_posts_path
+    expect(page).to have_content("ro")
```

Comment:
this should be a more unique string or the test can pass for a different reason (for example if it finds the text "from" which also has "ro")